src/util/metadata.ts

```diff
@@ -24,6 +24,7 @@ export interface ChainProperties {
  * safe to hardcode them.
  *
  * @ignore
+ * @todo Should we expose this publicly?
```

Comment:
Not sure, I think a lot of people hardcode them themselves. If it's been working without exposing then I think we can leave it.